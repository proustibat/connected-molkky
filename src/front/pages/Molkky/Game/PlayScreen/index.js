import React from 'react';

const PlayScreen = () => (
  <h1>PlayScreen</h1>
);

export default PlayScreen;
