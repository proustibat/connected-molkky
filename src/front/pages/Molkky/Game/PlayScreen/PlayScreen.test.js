import React from 'react';
import { shallow } from 'enzyme';
import PlayScreen from './index';

describe('PlayScreen', () => {
  it('should render the component correctly', () => {
    // Given / When
    const component = shallow(<PlayScreen />);

    // Then
    expect(component).toHaveLength(1);
    expect(component.find('h1')).toHaveLength(1);
  });
});
