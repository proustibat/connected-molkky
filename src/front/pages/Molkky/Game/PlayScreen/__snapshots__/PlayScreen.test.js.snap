// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayScreen should display the game over modal: modal game over props 1`] = `
Object {
  "loserIcon": [Function],
  "loserName": "Team Dog",
  "loserScore": 46,
  "onInit": [Function],
  "winnerIcon": [Function],
  "winnerName": "Team Cat",
  "winnerScore": 50,
}
`;
